### PR TITLE
Initial UI styles

### PR DESCRIPTION
**Pull request by conor-jh**
This PR updates some more prominent styles:

- Makes `RequestBar` an inline form with styled elements
- `RequestBar` gives user more feedback: slightly modified focused state, disabled button for invalid inputs, and alerts user if they try to submit with invalid inputs
- Empty state for no data is now styled
- `Button` component itself has more options for styling based on props

### Does not include

- Ability to expand/collapse panels
- Other text styling
- Functionality changes outside of request bar invalid input handling
- Style changes to data/tests

### CODE REVIEW

File: src/main/components/Button.jsx [hunk 1]
```diff
@@ -1,20 +1,36 @@
 import styled from 'styled-components';
 
+let backgroundColor;
+
 const Button = styled.button`
```
Comment (Donaldblanc) [inline src/main/components/Button.jsx:5]: This maybe for a new issue but the button appears to be fixed and does not adjust with the window size. When launching on my end,  the button span other panels.

File: src/main/components/RequestBar.jsx [hunk 2]
```diff
@@ -60,26 +71,27 @@ const RequestBar = (props) => {
...
+          && <Select name='method' id='fetchTypeInput' multiple={false} value={selected}
             onChange={handleChange} >
             <option value='GET'>GET</option>
-          </select>
+          </Select>
         }
         {
           (SourceOrDest === 'dest')
-          && <select name='method' id='fetchTypeInput' multiple={false} value={selected}
+          && <Select name='method' id='fetchTypeInput' multiple={false} value={selected}
```
Comment (Donaldblanc) [inline src/main/components/RequestBar.jsx:84]: may also be for another PR but  the options seem to still have the default styling, we should revisit post MVP.

**Review by Donaldblanc — APPROVED** [review]
Styling looks good, my only concern is when the app is on different screen sizes, are we going to have the same expected behavior.

**Review by RealJoelRivera — APPROVED** [review]
